tmt/steps/report/display.py

```diff
@@ -80,11 +82,17 @@ def _display_log_content(log: Path, shift: int) -> None:
         def display_outcome(result: BaseResult) -> None:
             """ Display a single result outcome """
 
-            if isinstance(result, CheckResult):
+            if isinstance(result, PhaseCheckResult):
+                self.verbose(f'{result.show()}', shift=PHASE_SHIFT + 1)
```

Comment:
This line should include the event, to distinguish between checks running before and after the given phase. It's in place for test-level checks, and it will be the case for phases as well, so the output should be similar.

Reply:
Good catch, thanks! I've added the event info.

```
/var/tmp/tmt/run-001

/plans/example
    report
        how: display
            pass /tests/call-dmesg
                skip dmesg (before-test check)
                skip dmesg (after-test check)
            fail /tests/mine-rep-multi
                skip dmesg (before-test check)
                skip dmesg (after-test check)
                pass /tests/mine-rep-multi/test/good
                fail /tests/mine-rep-multi/test/fail
                warn /tests/mine-rep-multi/test/weird
                    skip dmesg (before-test check)
                    skip dmesg (after-test check)
            pass /tests/mine-rep-pass
        summary: 2 tests passed and 1 test failed

total: 2 tests passed and 1 test failed
```